src/blog/understanding-ecmascript-part-1.md

```diff
@@ -35,7 +35,7 @@ To describe how `Object.prototype.hasOwnProperty` works, the spec uses pseudocod
 > 2. Let `O` be `? ToObject(this value)`.
 > 3. Return `? HasOwnProperty(O, P)`.
 
-…and…
+and
```

Comment:
Why remove these?

```suggestion
…and…	
```

Reply:
I think the ...-character is a weird artifact coming from the times when I edited the blog post draft in Google Docs (it changes ... into this weird ...-character). First I thought I'll replace with ... but it's cleaner without any unnecessary dots.

Reply:
AFAIK it's a character Mathias prefers and often adds instead of a regular `...` after PRs have been merged, so it might be where it's coming from :)